packages/react/src/exampleTypes/ResultTable/index.js

```diff
@@ -122,8 +132,17 @@ let ResultTable = ({
             (columnGroupRow, i) => (
               <Tr key={i}>
                 {F.mapIndexed(
-                  ({ groupName, colspan, HeaderCell = Th, HeaderGroup }, j) => (
-                    <HeaderCell key={j} colSpan={colspan}>
+                  (
+                    {
+                      groupName,
+                      colspan,
+                      headerCellProps,
+                      HeaderCell = Th,
+                      HeaderGroup,
+                    },
+                    j
+                  ) => (
+                    <HeaderCell key={j} colSpan={colspan} {...headerCellProps}>
                       <span>
```

Comment:
Shouldn't this span be removed also here?

Reply:
Indeed. Thanks for catching that 👍